.gitignore

```diff
@@ -114,6 +114,7 @@ celerybeat.pid
 
 # Environments
 .env
+.env.example
```

Comment:
you should actually change this to ```.env.*```. gitignore can have regex matches. read what regex is and what * means.

Reply:
Also, keep better names for the files. ```.example``` doesn't tell me what the file contains. Keep better names.